packages/taquito-rpc/src/types.ts

```diff
@@ -191,6 +221,14 @@ export interface InlinedPreEndorsement {
 
 export type BallotVote = 'nay' | 'yay' | 'pass';
 
+export interface OperationContentsAttestation {
+  kind: OpKind.ATTESTATION;
+  slot: number;
```

Comment:
Should `slot` and `round` be required or optional?

Reply:
both required, here the openAPI reference 

```
      "018-Proxford.operation.alpha.operation_contents_and_result": {
        "oneOf": [
          {
            "title": "Attestation",
            "type": "object",
            "properties": {
              "kind": {
                "type": "string",
                "enum": [
                  "attestation"
                ]
              },
              "slot": {
                "type": "integer",
                "minimum": 0,
                "maximum": 65535
              },
              "level": {
                "type": "integer",
                "minimum": -2147483648,
                "maximum": 2147483647
              },
              "round": {
                "type": "integer",
                "minimum": -2147483648,
                "maximum": 2147483647
              },
              "block_payload_hash": {
                "$ref": "#/components/schemas/value_hash"
              },
              "metadata": {
                "type": "object",
                "properties": {
                  "balance_updates": {
                    "$ref": "#/components/schemas/018-Proxford.operation_metadata_with_legacy_attestation_name.alpha.balance_updates"
                  },
                  "delegate": {
                    "$ref": "#/components/schemas/Signature.Public_key_hash"
                  },
                  "consensus_power": {
                    "type": "integer",
                    "minimum": -1073741824,
                    "maximum": 1073741823
                  },
                  "consensus_key": {
                    "$ref": "#/components/schemas/Signature.Public_key_hash"
                  }
                },
                "required": [
                  "delegate",
                  "consensus_power",
                  "consensus_key"
                ]
              }
            },
            "required": [
              "kind",
              "slot",
              "level",
              "round",
              "block_payload_hash",
              "metadata"
            ]
          },
```